views/admin.php.inc

```diff
@@ -1,183 +1,173 @@
-                            </div>
-                        </div>
-                        <div class="form-group col-lg-12">
-                            <label for="show_oncustom" class="col-lg-6 control-label">Show on pages:</label>
-                            <div class="col-lg-6 control-label">
-                                <?php if (isset($display_opts->show_oncustom) && !empty($display_opts->show_oncustom)) : ?>
-                                    <?php $whitelist = json_decode($display_opts->show_oncustom) ?>
-                                    <textarea class="form-control show_specific" name="show_oncustom"
-                                        id="show_oncustom" cols="30" rows="10"><?php foreach ($whitelist as $page) { echo $page."\r\n"; } ?></textarea>
-                                <?php else : ?>
-                                    <textarea class="form-control show_specific" name="show_oncustom" id="show_oncustom" cols="30" rows="10"></textarea>
-                                <?php endif; ?>
-                                <br>
-                                <p style="text-align: justify;">
-                                Add URLs to pages in which you would like to show the widget.<br>
-                                Put each URL in a new line.
-                                </p>
+{   ob_start();
+    ?><script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+    <div id="content" >
+        <div class="l-wrapper">
+            <div class="l-wrapper-inner container container-fluid">
+                <div class="l-messages" role="status" aria-label="Status messages" id="optionsSuccessMessage" style="display: none;">
+                    <div class="messages status">
+                        <h2 class="element-invisible">Status message</h2>
+                        Successfully set widget options to your site
+                    </div>
+                </div>
+                <div class="l-page-title">
+                    <a id="main-content"></a>
+                </div>
+                <div class="l-content" role="main" aria-label="Main content">
+                    <div>
+                        <fieldset class="form-wrapper" id="property-widget-settings">
+                            <legend><span class="fieldset-legend">Property and Widget Selection</span></legend>
+                            <div class="fieldset-wrapper">
+                                <div class="fieldset-description">Select the property and widget from your tawk.to account</div>
+                                <?php if (!$sameUser) { ?>
+                                    <div id="widget_already_set" class="alert alert-warning">Notice: Widget already set by other user</div>
+                                <?php } ?>
+                                <iframe id="tawkIframe" src="" style="min-height: 275px; width: 80%; border: none; margin: 5px 0; padding: 10px; background: #FFF;"></iframe>
```

Comment:
`padding` and `background` can be removed. Then, `width` can be set to `100%`.

Reply:
Done